polaris-react/src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -40,14 +43,18 @@ export function ActionMenu({
 
   return (
     <div className={actionMenuClassNames}>
```

Comment:
Not a fan of this change but couldn't come up with a better way of doing it. Open to suggestions